src/__tests__/parser.spec.ts

```diff
@@ -120,4 +120,38 @@ describe('parser', () => {
         assert.equal(true, i.members[3].isRequired);
     });
 
-});
\ No newline at end of file
+    it('Should should avoid exported objects', () => {
```

Comment:
I think this test name should be different. It doesn't seem to explain what test does.